dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -918,3 +918,15 @@ private fun VitalInfo.toPerformanceMetric(): ViewEvent.FlutterBuildTime {
         average = meanValue
     )
 }
+
+private fun VitalInfo.toInversePerformanceMetric(): ViewEvent.FlutterBuildTime {
+    return ViewEvent.FlutterBuildTime(
+        min = invertValue(maxValue),
```

Comment:
why are you switching `min` with `max` here ?

Reply:
Maybe a comment to explain that would be nice here. It will be very hard to remember why we are doing this.

Reply:
I got that...what is hard for me to understand is that right now we are inverting the `max` value to create the `min`. But is this relevant ?